include/zephyr/net/socket_ncs.h

```diff
@@ -121,6 +112,26 @@ extern "C" {
  *  option before the next send call.
  */
 #define SO_RAI_WAIT_MORE 54
+
+/* NCS specific IPPROTO_ALL level socket options */
+
+/** IPv4 and IPv6 protocol level (pseudo-val) for nRF sockets. */
+#define IPPROTO_ALL 512
+/** sockopt: disable all replies to unexpected traffics */
+#define SO_SILENCE_ALL 30
+
+/* NCS specific IPPROTO_IP level socket options */
+
+/** sockopt: disable IPv4 ICMP replies */
+#define SO_IP_ECHO_REPLY 31
```

Comment:
Non-zero enables ICMP echo replies on IPv4. Perhaps this should be "enable IPv4 ICMP replies"? (same goes for the IPv6 socket option)

Reply:
Changed: disable -> enable